source/documentation/common/upgrade/proc-Upgrading_the_Manager_to_4-4.adoc

```diff
@@ -62,10 +63,33 @@ ifdef::local_database_upgrade,remote_database_upgrade[. Log in to the {engine-na
 ----
 # engine-backup --scope=all --mode=backup --file=backup.bck --log=backuplog.log
 ----
-
 . Copy the backup file to a storage device outside of the {virt-product-shortname} environment.
 
 ifdef::SHE_upgrade[]
+
+. Shut down the self-hosted engine.
++
+[options="nowrap" subs="normal"]
+----
+# poweroff
+----
++
+[NOTE]
+====
+Before you shut down the self-hosted engine virtual machine, write down the mac address of the self-hosted engine network interface if you wish to reuse it for the {virt-product-fullname} {engine-name} 4.4 deployment.
```

Comment:
Suggest you replace "wish" with want to conform to IBM style.

Reply:
thanks